### PR TITLE
Nav fix for EI 11

### PR DESCRIPTION
**Pull request by DerekSgov**
## Summary
Fix for the Nav issues on EI 11, using <script src="https://cdn.polyfill.io/v2/polyfill.min.js" /> (with helmet tags) in the layout.js file

## Notable Changes 
-
-
-

### CODE REVIEW

File: app-web/src/hoc/Layout.js [hunk 1]
```diff
@@ -27,6 +28,9 @@ export const Layout = ({ children }) => {
 
   return (
     <StyledContainer fluid>
+      <Helmet>
+        <script src="https://cdn.polyfill.io/v2/polyfill.min.js" />
```
Comment (patricksimonian) [inline app-web/src/hoc/Layout.js]: lets add a comment here explaining this is to fix some issues with running in ie 11

Comment (DerekSgov) [reply app-web/src/hoc/Layout.js]: done!

**Comment by patricksimonian** [comment]
@DerekSgov remind about **rebase interactive mode**

**Comment by patricksimonian** [comment]
we'll fix up that last commit

**Comment by patricksimonian** [comment]
@DerekSgov this is gona need rebasing again, doubel check with master to ensure this PR is up to date before promoting to test

**Pull request merged**